src/madhuraMJun20/scannerClass/PalindromeNumber.java

```diff
@@ -0,0 +1,38 @@
+package madhuraMJun20.scannerClass;
+
+/*Verify given number is palindrome or not. 
+Input: number = 45654
+Output : Given number 45654 is palindrome.
+
+Input: number = 56789
+Output : Given number 56789 is not palindrome.
+*/
+public class PalindromeNumber {
+	int number;
```

Comment:
No need of this instance variable "number".

Reply:
Done